src/librustc_resolve/lib.rs

```diff
@@ -68,7 +72,8 @@ use syntax::ast::{ExprLoop, ExprWhile, ExprMethodCall};
 use syntax::ast::{ExprPath, ExprStruct, FnDecl};
 use syntax::ast::{ForeignItemFn, ForeignItemStatic, Generics};
 use syntax::ast::{Ident, ImplItem, Item, ItemConst, ItemEnum, ItemExternCrate};
-use syntax::ast::{ItemFn, ItemForeignMod, ItemImpl, ItemMac, ItemMod, ItemStatic, ItemDefaultImpl};
+use syntax::ast::{ItemFn, ItemForeignMod, ItemImpl, ItemMac, ItemMod,
+                  ItemStatic, ItemDefaultImpl};
```

Comment:
I don't know, reordering the input tokens is probably going too far, IMO ... usually pretty print / format tools just focus on adjusting whitespacing and line breaks.

(As another example, does `rustfmt` ever insert or delete trailing commas in argument/field lists?  I actually do not know the answer)